zookeeper-client/zookeeper-client-c/src/zookeeper.c

```diff
@@ -344,6 +344,38 @@ static int is_sasl_auth_in_progress(zhandle_t* zh)
 #endif /* HAVE_CYRUS_SASL_H */
 }
 
+/*
+ * Extract the type field (ZOO_*_OP) of a serialized RequestHeader.
+ *
+ * (This is not the most efficient way of fetching 4 bytes, but it is
+ * currently only used during SASL negociation.)
```

Comment:
typo: negociation  -> negotiation
here and in other places

Reply:
Thanks; should be fixed now!

Reply:
makes sense